### PR TITLE
New medicine: Naloxone

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: Cyberboss
add: New medicine: Naloxone. This drug prevents overdoses of all other reagents as long as it's in your system and can cure overdoses if applied in less than 10s. But, be careful, getting addicted or overdosing on this will have the same effect on all other reagents. Recipe: 4uO + 2uH + 1uN = 2u. -Designed by ma44
/:cl:

- [x] Needs a NEW recipe: @ma44

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1116,3 +1116,81 @@
 	M.adjustToxLoss(2, 0)
 	. = 1
 	..()
+
+/datum/reagent/medicine/naloxone
+	name = "Naloxone"
+	id = "naloxone"
+	description = "Catalyzes and stabilized other reagents in the bloodstream to prevents overdoses. Does not prevent overdosing on itself or stop previous overdoeses."
```
Comment (coiax) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: to _prevent_ overdoses

**Comment by coiax** [comment]
But this doesn't stop overdoses in progress, right?

**Comment by Cyberboss** [comment]
Nope

**Comment by ma44** [comment]
Increase metabolization rate to 0.5 per tick (from 0.4)

Recipe should be 4 oxygen, 2 hydrogen and 1 nitrogen, combining all of them together should result in 2 units of it

**Comment by PKPenguin321** [comment]
Wait so if it doesn't stop overdoses in progress then what happens if you take a pill with this and another chem that you OD on at the same time

**Comment by Cyberboss** [comment]
Good point, I'll make it stop overdoses under 10 cycles

**Comment by swindly** [comment]
Hydrogen and Nitrogen makes Ammonia.

**Comment by Cyberboss** [comment]
@ma44

**Comment by ma44** [comment]
Reeeeeee
3 Oxygen, 1 hydrogen and 1 ammonia for 2 units

**Comment by Shapsy** [comment]
What is the intended use for this?

**Comment by kevinz000** [comment]
Make it slightly harder to use so I don't get more chemical conflicts.

**Comment by AdamElTablawy** [comment]
over

powered

**Comment by ma44** [comment]
Please elaborate

**Comment by Iamgoofball** [comment]
# stop QoLing away all my anti abuse features

**Comment by juanototo2** [comment]
>goof is mad chem doesn't get any love compared to RnD and lavaland

10/10

Last OP mix chemistry had was polyacid and clf3, and that shit was from 2007.

Now RnD has shit that could outrobust pulse pistols and mining can turn you into **MEGAFAUNA**

**Comment by Iamgoofball** [comment]
>giving chem love by removing an important balance feature

**Comment by juanototo2** [comment]
>giving atmos love by giving the server-erasing Pete-rivaling gas that is memeon

>giving sec love by giving them instant burst fire instamurder disablers and pocket ion weapons

You could say the same thing about almost all of your code-based shitposts.

**Comment by PKPenguin321** [comment]
goof all this does is allow careful chemists to maybe add 20 to the max OD threshold of other chems, which wont even stop addiction

**Comment by Haevacht** [comment]
I hate this, but I'm with goof. OD is a balancing mechanic, this is blatantly power creep for chemists, letting them make stronger bullshit pills.

**Comment by Iamgoofball** [comment]
@juanototo2 Hang yourself.

**Comment by ma44** [comment]
@Cyberboss Change metabolism to 1 reagent per cycle
Amount of Naloxone should be changed to 1 from 2
OD threshold to 10 units from 30u 

Considering that this is a "early" game chem, it should also be very taxing on the chem dispenser as well as metabolism. 10 units of Naloxone will last for 20 seconds with current metabolism rates and will also require 6 energy for said 10 units.

**Comment by ma44** [comment]
Also your assuming your going to put enough units of chemicals to overdose on, while drinking naloxone and making it last long enough for it to matter

**Comment by KorPhaeron** [comment]
The balance implications of this aren't really that bad I don't think but this really does feel like another "quality of life" to undo a purposeful limitation

**Pull request closed without merging**